Makefile

```diff
@@ -42,6 +42,27 @@ endif
 build:
 	$(GO) build $(GO_LDFLAGS) $(BUILDV) $(BUILDTAGS) -o $(PKGNAME) $(BUILDFILES)
 
+PLATFORMS := darwin linux windows
+
+WIN_FLAGS = -ldflags "-X main.Version=$(VERSION) -linkmode=internal -s -w -extldflags '-static' -extld=x86_64-w64-mingw32-gcc"
+
+buildNatives:
+	@for platform in $(PLATFORMS); \
+	do \
+		arch="amd64"; \
+		output_name="$(PKGNAME)-$$platform"; \
+		if [ $$platform = windows ]; then \
+			echo $$output_name; \
+			echo "asdasddassad"; \
```

Comment:
Mean only for testing mayhaps?

Reply:
argh, I though I made that afterwards should be "gone in 60 seconds"